### PR TITLE
Add test cases for replication filters

### PR DESCRIPTION
**Pull request by pasin**
* Added replication-filter test cases for documentIDs, channels, pushFilter, and pullFilter.
* Updated travel-sample dataset to include info about channels

### CODE REVIEW

File: spec/tests/002-replication-filter.md [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+         * pushFilter:
+            * name: `documentIDs`
+            * params: `{ "documentIDs": { "inventory.airline": "airline_10", "airline_22", "airline_1x"} }`
+      * `inventory.route`
+         * pushFilter:  
+            * name: `deletedDocumentsOnly`
+            * params: `{}`
+    * endpoint: `/travel-sample-inventory`
+    * type: push
+    * continuos: false
```
Comment (borrrden) [inline spec/tests/002-replication-filter.md:131]: typo

File: spec/tests/002-replication-filter.md [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+
+### Steps
+
+1. Load `travel-sample` dataset into a database.
+2. Start a replicator: 
+    * collections : 
+      * `inventory.airline`
+         * pushFilter:
+            * name: `documentIDs`
+            * params: `{ "documentIDs": { "inventory.airline": "airline_10", "airline_22", "airline_1x"} }`
```
Comment (borrrden) [inline spec/tests/002-replication-filter.md:124]: If this filter is specific to this collection, is it necessary to include the collection in the JSON?  Also the JSON is invalid.

File: spec/tests/002-replication-filter.md [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+
+### Steps
+
+1. Load `travel-sample` dataset into a database.
+2. Start a replicator: 
+    * collections : 
+      * `inventory.airport`
+         * pullFilter:
+            * name: `documentIDs`  
+            * params : `{ "documentIDs": {"inventory.airport": "airport_1254", "airport_1260", "airport_1x"} }`
```
Comment (borrrden) [inline spec/tests/002-replication-filter.md:155]: Same comment as push

File: spec/tests/002-replication-filter.md (hunk 2 above)
Comment (pasin) [reply spec/tests/002-replication-filter.md:124]: Not sure what I was thinking. It should be just an array of doc ids.

**Review by borrrden — APPROVED** [review]

**Pull request merged**